config/initializers/sidekiq.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative '../../app/lib/sidekiq_error_handler'
```

Comment:
Uh, why is that necessary? Shouldn't it get auto-loaded when mentioning `SidekiqErrorHandler`?

Reply:
Because Rails complains about autoloading during initialization

Reply:
ok. I would move outside `app` then, to `lib/`

Reply:
looks good to me if this is moved to `lib/`